tests/unit/services/cookies-test.js

```diff
@@ -74,6 +74,11 @@ describe('CookiesService', function() {
         expect(afterRoundtrip).to.eq(value);
       });
 
+      it('handles invalid values for cookies', function() {
+        document.cookie = '=blank';
+        expect(this.subject().read('')).to.be.blank;
```

Comment:
That's because of [this line](https://github.com/simplabs/ember-cookies/blob/master/addon/services/cookies.js#L64). I guess you'll want to assert it returns an object that does not contain an empty key then.